presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -121,7 +125,7 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
                 handle.getConstraint(),
                 OptionalLong.of(limit));
 
-        return Optional.of(new LimitApplicationResult<>(handle, jdbcClient.isLimitGuaranteed()));
+        return Optional.of(new LimitApplicationResult<>(handle, true));
```

Comment:
Did this lose the ability to say "limit can't be guaranteed"?

Reply:
It did -- consciously, but I understood this isn't a good idea. Will update.